python-sdk/tutorials/nuscenes_lidarseg_tutorial.ipynb

```diff
@@ -363,6 +387,44 @@
     "#                                                    out_path=os.path.expanduser('~/Desktop/my_rendered_scene.avi'))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualizing LIDAR segmentation predictions\n",
+    "In all the above functions, the labels of the LIDAR pointcloud which have been rendered are the ground truth. If you have trained a model to segment LIDAR pointclouds and have run it on the nuScenes-lidarseg dataset, you can visualize your model's predictions simply by using the `show_lidarseg_preds` argument in the above functions.\n",
+    "\n",
+    "You simply need to pass the path to the folder where your predictions are stored into `show_lidarseg_preds`, like so:"
```

Comment:
Explain that here we will do it with the ground-truth, but that the files are identical.